test/integration/data-types.test.js

```diff
@@ -478,8 +478,10 @@ describe(Support.getTestDialectTeaser('DataTypes'), () => {
     const Model = this.sequelize.define('user', {
       stamp: Sequelize.DATEONLY
     });
+
     const testDate = moment().format('YYYY-MM-DD');
     const newDate = new Date();
+    const dateOnly = moment(newDate).format('YYYY-MM-DD');
```

Comment:
No need to create a new variable, you can just use `testDate`

Reply:
The original test seems to be two in one. I believe the original intent of the test was to not only show that the result is a 'YYYY-MM-DD' string, but also to show how a Date object is processed when passed into a DATEONLY field. `dateOnly` represents the expected string when a Date object is passed and is then retrieved from the database client. May be better to split the test?